src/test/java/org/prebid/server/it/OwnAdxTest.java

```diff
@@ -0,0 +1,37 @@
+package org.prebid.server.it;
+
+import io.restassured.response.Response;
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.prebid.server.model.Endpoint;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.io.IOException;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalToJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static java.util.Collections.singletonList;
+
+@RunWith(SpringRunner.class)
+public class OwnAdxTest extends IntegrationTest {
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromOwnAdx() throws IOException, JSONException {
+        // given
+        WIRE_MOCK_RULE.stubFor(post(urlPathEqualTo("/ownadx-exchange/bid//"))
```

Comment:
@SerhiiNahornyi isn't that weird URL? I don't understand why the json requires only one of those params while the URL needs all three

I also don't see any example on the GO side when they cover the case of now having at least one of those params

Reply:
I thinking about it, I suppose owned.json should contain those params in one `required` block.

Apart that GO has differend naming between ownadx.json and imp_ownadx.go.

Reply:
https://github.com/prebid/prebid.github.io/pull/4687/files describes all the fields as mandatory... I suppose it's a mistake on the GO side, but it's worth discussing with @SerhiiNahornyi

Reply:
Please point this question to @bretg, json schema definitely needs to be updated
So we need to reach OwnAdx

Reply:
@marki1an any updates here?